dockci/models/job.py

```diff
@@ -57,5 +57,7 @@ def _all_builds(self):
     slug = None
     repo = LoadOnAccess(default=lambda _: '')
     name = LoadOnAccess(default=lambda _: '')
+    hipchat_api_token = LoadOnAccess(default=lambda _: '')
```

Comment:
sorry, i mean a # TODO as in in the code like
```python
# TODO encrypt etc etc
hipchat_api_token = LoadOnAccess(default=lambda _: '')
```

pylint picks these up and lists them for you, so it's easy to find in the code